### ISSUE TITLE
Training epochs

### ISSUE THREAD

**Issue by hellojialee** [xingyizhou/CenterNet#775]
Do we really need to train the pre-trained network such as Hourglass-104 for so many epochs? 

I see you continue to train the Hourglass-104 backbone which is used as the detection task on COCO for 150 epochs on the pose estimation track with Adam optimizer. But I fine-tune on my own and notice that the loss is quickly to be consistent after 60 epoch (decrease the LR two time from 2.5e-4 every 15 epochs).

**Comment by xingyizhou** [comment]
Thank you for bringing this up! I didn't play with Hourglass after the submission. For DLA, my observation is training 70 epochs is mostly sufficient.

**Comment by hellojialee** [comment]
Thank you for your comment!
